### PR TITLE
String manipulation 1

### PR DESCRIPTION
**Pull request by Boyul-Kim**
image

### CODE REVIEW

File: string-manipulation-1/src/is-vowel.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+function isVowel(char) {
+  var charLower = char.toLowerCase();
+  var vowel = ['a', 'e', 'i', 'o', 'u'];
+
+  if (vowel[0] === charLower) {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:6]: Could you do this with a loop?

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+function isUpperCased(word) {
+  var falseCount = 0;
+  for (var i = 0; i <= word.length - 1; i++) {
+    if (word[i].toUpperCase() !== word[i]) {
+      falseCount++;
+    }
+  }
+
+  if (falseCount > 0) {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-upper-cased.js:10]: Why the count? Can't you just return as soon as you hit a false in your loop?

File: string-manipulation-1/src/is-lower-cased.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+function isLowerCased(word) {
+  var falseCount = 0;
+  for (var i = 0; i <= word.length - 1; i++) {
+    if (word[i].toLowerCase() !== word[i]) {
+      falseCount++;
+    }
+  }
+
+  if (falseCount > 0) {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-lower-cased.js:10]: Same here, you can do this without a count 👍

File: string-manipulation-1/src/is-vowel.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    return true;
+  } else if (vowel[2] === charLower) {
+    return true;
+  } else if (vowel[3] === charLower) {
+    return true;
+  } else if (vowel[4] === charLower) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (uzair-ashraf) [inline string-manipulation-1/src/is-vowel.js:18]: You want to use a loop for this.

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/get-words.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function getWords(string) {
+
+  if (string === '') {
+    return [];
+  } else if (getWords.length > 0) {
```
Comment (thebearingedge) [inline string-manipulation-1/src/get-words.js:6]: This can just be an `else`.

File: string-manipulation-1/src/is-lower-cased.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+function isLowerCased(word) {
+  var falseCount = 0;
+  for (var i = 0; i <= word.length - 1; i++) {
+    if (word[i].toLowerCase() !== word[i]) {
+      falseCount++;
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-lower-cased.js:6]: you can just `return false` here.

File: string-manipulation-1/src/is-lower-cased.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    }
+  }
+
+  if (falseCount > 0) {
+    return false;
+  } else {
+    return true;
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-lower-cased.js:14]: If your loop doesn't `return false`, you can `return true` here.

File: string-manipulation-1/src/is-upper-cased.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    return false;
+  } else {
+    return true;
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/is-upper-cased.js:14]: Same as your `isLowerCased`, a counter isn't really necessary.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**